SoftLayer/CLI/bandwidth/pools_create.py

```diff
@@ -21,13 +21,36 @@
     "FRA": "FRA"
 }
 
+regions = ['SJC/DAL/WDC/TOR/MON', 'AMS/LON/MAD/PAR', 'SNG/HKG/OSA/TOK', 'SYD', 'MEX', 'SAO', 'CHE', 'MIL', 'SEO', 'FRA']
+HELP_STMT = "Ex: 'HKG' or 'SNG/HKG/OSA/TOK'"
+
+
+def check_region_param(ctx, param, value):
+    """Check if provided region is region group or part of region"""
+
+    # :params string value: Region or Region-Groups
+    # return string Region-Groups
+
+    _ = [ctx, param]
+    region_group = None
+    for key in location_groups:
+        if value in key or value is key:
+            region_group = key
+        else:
+            continue
+
+    if region_group:
+        return region_group
+    else:
+        raise click.BadParameter(f"{value} is not a region or part of any region.\
+                                 \nAvailable Choices: \033[1;32m{regions}")
+
 
 @click.command(cls=SLCommand)
 @click.option('--name', required=True, help="Pool name")
 @click.option('--region', required=True,
-              type=click.Choice(['SJC/DAL/WDC/TOR/MON', 'AMS/LON/MAD/PAR', 'SNG/HKG/OSA/TOK',
-                                'SYD', 'MEX', 'SAO', 'CHE', 'MIL', 'SEO', 'FRA']),
-              help="Region selected")
+              help=f"Choose Region/Region-Group {regions}", callback=check_region_param)
+@click.help_option('--help', '-h', help=f"Specify Region or Region group - \033[1;32m{HELP_STMT}")
 @environment.pass_env
 def cli(env, name, region):
     """Create bandwidth pool."""
```

Comment:
```suggestion
    """Create bandwidth pool.
    
    Region can be the full zone name 'SJC/DAL/WDC/TOR/MON', or just a single datacenter like 'SJC'.

    Example::

        slcli bandwidth pool-create --name testPool --region DAL
        slcli bandwidth pool-create --name testPool --region SJC/DAL/WDC/TOR/MON
```

`Example::` is a format that will get picked up by the documentation generator to make it a nice formatted table, like https://softlayer-python.readthedocs.io/en/latest/cli/vs/#virtual-bandwidth